cardano-cli/test/Test/Golden/TxView.hs

```diff
@@ -16,11 +16,39 @@ txViewTests :: IO Bool
 txViewTests =
   checkSequential $
     Group "`transaction view` Goldens"
-      [ ("golden_view_shelley", golden_view_shelley)
+      [ ("golden_view_byron",   golden_view_byron)
+      , ("golden_view_shelley", golden_view_shelley)
       , ("golden_view_allegra", golden_view_allegra)
       , ("golden_view_mary",    golden_view_mary)
+      -- , ("golden_view_alonzo",  golden_view_alonzo)
```

Comment:
Is this not ready to be enabled?

Reply:
I want to make it in a separate patch. This one is overloaded already.